### ISSUE TITLE
External links not working

### ISSUE THREAD

**Issue by p-jo** [keybits/permanote#4]
I'm having problems with the links I write in markdown, I think it's because it's trying to open them in the Sandstorm iframe, so they are blocked in the browser. I get this in the console:

`Refused to frame 'http://www.example.com' because it violates the following Content Security Policy directive: "frame-src https://*.myurl.sandcats.io".`

**Comment by rainbowjaw** [comment]
if you open in a new tab (middle click, or from menu)  you can get around this.  For me it's only a problem if i try to open the link in the same window as sandstorm.

**Comment by ocdtrekkie** [comment]
Yeah, that's exactly the case. Ideally, Sandstorm app packages should ensure external links are set to open in new tabs, but some are missing this fix.
